app/components/tableHeading.js

```diff
@@ -0,0 +1,14 @@
+export function TableHeading() {
+  return (
+    <thead>
+      <tr className='table-dark'>
+        <th scope='col' className="col-2">Picture</th>
+        <th scope='col' className="col-2">Name</th>
+        <th scope='col' className="col-2">Email</th>
+        <th scope='col' className="col-2">Phone Number</th>
+        <th scope='col' className="col-1"></th>
+        <th scope='col' className="col-1"></th>

```

Comment:
DRY - don't repeat your self, 
What about mapping a fixed const of elements, with their own col:
const columns = [{ className: 'col-2', value: Picture }..... ]
```suggestion
columns.map((column) =>  <th scope='col' className="column.className">column.value</th>)
```